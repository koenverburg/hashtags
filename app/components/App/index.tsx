import React from 'react'

export function App() {
  return (
    <section>
      <h1>Hiii static site gen</h1>
    </section>
  )
}
