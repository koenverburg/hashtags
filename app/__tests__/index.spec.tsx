/* tslint:disable no-implicit-dependencies */
import React from 'react'
import App from '../pages/index'
import { shallow, ShallowWrapper } from 'enzyme'

describe('App component', () => {
  let wrapper: ShallowWrapper

  beforeAll(() => {
    wrapper = shallow(<App />)
  })

  it('render', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
