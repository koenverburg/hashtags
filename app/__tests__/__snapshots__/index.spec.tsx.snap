// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component render 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-md-8"
    >
      <Memo(wrappedComponent) />
    </div>
  </div>
</div>
`;
